### PR TITLE
Add customizable ports

### PR DESCRIPTION
**Pull request by jderusse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +13,34 @@ Usage
...
 Hosted on Docker Hub: https://hub.docker.com/r/acmephp/testing-ca/
 
+Start the boulder container in background.
+
+```bash
+docker run -d --net host acmephp/testing-ca
+```
+
+> By design, to test the domain, boulder will resolve the domain to 127.0.0.1,
+and call the given URL `http://www.acmephp.com:5002/.well-known/acme-challenge/${TOKEN}`.
```
Comment (tgalopin) [inline README.md]: What is `acmephp.com`?

**Comment by tgalopin** [comment]
That's great, thanks!

**Comment by tgalopin** [comment]
+1

**Comment by jderusse** [comment]
Fixes #2

**Pull request merged**